### PR TITLE
Add Recency Aware Collaborative Filtering for Next Basket Recommendation (UPCF) Model

### PR DESCRIPTION
**Pull request by lthoang**
### Related Issues

https://github.com/PreferredAI/cornac/issues/579

### Checklist:

- [ ] I have added tests.
- [x] I have updated the documentation accordingly.
- [x] I have updated `README.md` (if you are adding a new model).
- [x] I have updated `examples/README.md` (if you are adding a new example).
- [ ] I have updated `datasets/README.md` (if you are adding a new dataset).

### CODE REVIEW

File: cornac/models/upcf/recom_upcf.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        recency=1,
+        locality=1,
+        asymmetry=0.25,
+        verbose=False,
+    ):
+        super().__init__(name=name, trainable=False, verbose=verbose)
+        self.recency = recency
+        self.locality = locality
+        self.asymmetry = asymmetry
+        self.seed = seed
```
Comment (tqtg) [inline cornac/models/upcf/recom_upcf.py:66]: `seed` is not available

File: cornac/models/upcf/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+similaripy
```
Comment (tqtg) [inline cornac/models/upcf/requirements.txt:1]: I'm not sure if this library is well-maintained and will be backward compatible for future versions. Maybe we can pin a specific working version?

File: cornac/models/upcf/recom_upcf.py (hunk 1 above)
Comment (lthoang) [reply cornac/models/upcf/recom_upcf.py]: Updated

File: cornac/models/upcf/requirements.txt (hunk 2 above)
Comment (lthoang) [reply cornac/models/upcf/requirements.txt]: Updated.

**Review by tqtg — APPROVED** [review]

**Pull request merged**